### DISCUSSION TITLE
Come on, tomorrow will be better

### DISCUSSION THREAD

**Discussion by Dr-chen99** [community/community#19876] [Other Feature Feedback, Questions, & Ideas]
Come on, tomorrow will be better

**Comment by d-coder111** [comment]
Definitely😍 It will be bright and better!!!

Hey Dr-chen99, Welcome to the github community👋
Wishing u good luck for your amazing journey ahead and let's connect on github to learn together😊🤝

**Reply by YouthguyZ** [reply]
> 
d-coder111 let's connect on github to learn together too follow me🍀

**Reply by d-coder111** [reply]
sure sure😊🤝

**Reply by Dr-chen99** [reply]
sure sure

**Comment by YouthguyZ** [comment]
yes guy keep calm and carry on. good luck to you👏🏻👏🏻

**Comment by pr0phet87** [comment]
Tx?
